### PR TITLE
Make no_screenshot image more clear

### LINKED ISSUE #12344: Provide default thumbnail for content that have none
## Problem

immagine

## Solutions
Put a default thumbnail (slightly lighter colour + question mark?)

### PR DESCRIPTION
**Pull request by Zughy**
Fixes #12344 

## To do

This PR is Ready for Review.

## How to test
eyes

Before:
immagine

immagine

After:
immagine

immagine

### CODE REVIEW

**Comment by sfan5** [comment]
Don't you need to add it to some license.txt file for attribution?

**Comment by Zughy** [comment]
@sfan5 done

**Review by sfan5 — APPROVED** [review]

**Comment by NathanSalapat** [comment]
Wouldn't a 'no screenshot' image make more sense? I can see a question mark confusing users.
This looks like Minetest is unsure of the content, or that something is wrong with the content.

**Comment by rubenwardy** [comment]
I prefer the current image, or just a flat color. Having an image with less visual noise grabs attention less

**Comment by Zughy** [comment]
@NathanSalapat Yes, I've tried, but it was too much detailed to be done in pixel art.

@rubenwardy no offence, but current menu is to redo completely, that image is not the main problem. We could start with dark blue mod names on a black background. Anyway, this is just a way to me to close an issue I've opened for #12345 . I'm not planning to spend much time on this, so if this became controversial along the way, I'll just close it

**Comment by Warr1024** [comment]
> Wouldn't a 'no screenshot' image make more sense?

Aside from personal preferences, having any text in the image would bring translations into scope.  Presumably that could complicate the layout, i.e. if you have to use a label to make it translatable.

**Comment by Warr1024** [comment]
It may be worth considering just copying ContentDB's solution to this same problem.

https://github.com/minetest/contentdb/blob/master/app/public/static/placeholder.png

**Comment by rollerozxa** [comment]
+1 for the ContentDB no screenshot icon, although I would like it a bit more zoomed in... so something like this:

no_screenshot
Screenshot_20220519_095217

**Comment by sfan5** [comment]
> @sfan5 what do you think?

sure

**Comment by Zughy** [comment]
@rubenwardy there, you can approve it now :D

**Comment by sfan5** [comment]
Texture license?

**Review by nerzhul — APPROVED** [review]

File: LICENSE.txt [hunk 1]
```diff
@@ -63,6 +63,9 @@ Zughy:
 
 appgurueu:
   textures/base/pack/server_incompatible.png
+  
+erlehmann, rollerozxa, Warr1024:
+textures/base/pack/no_screenshot.png
```
Comment (sfan5) [inline LICENSE.txt:68]:
```suggestion
  textures/base/pack/no_screenshot.png
```

File: LICENSE.txt [hunk 2]
```diff
@@ -63,6 +63,9 @@ Zughy:
 
 appgurueu:
   textures/base/pack/server_incompatible.png
+  
+erlehmann, rollerozxa, Warr1024:
```
Comment (rubenwardy) [inline LICENSE.txt:67]: Should probably be in order of change

```suggestion
erlehmann, Warr1024, rollerozxa:
```

**Review by rubenwardy — APPROVED** [review]

**Comment by appgurueu** [comment]
Triple approval?

**Pull request merged**